Digital 1/Assets/Scripts/UI/HUD/Crosshair.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Crosshair : MonoBehaviour
+{
+    public GameObject canvas;
+    public bool visible;
+    public bool spinning;
+    public float degPerSecond;
+    public int chargeButton;
+    public float chargePerSecond;
+    public float maxCharge;
+    public Color defaultColor;
+    public Color chargeColor;
+    public float retractionTime;
```

Comment:
These should all be private, with some having [SerializeField] in case you are setting some of them in the inspector. The ones that are changed outside of the class should have properties for public accessibility.